### PR TITLE
[Help Text] fix mri violations help text (markdown version) 

### PR DESCRIPTION
**Pull request by llevitis**
This PR updates the Markdown help text for the MRI Violations module.

### CODE REVIEW

File: modules/mri_violations/help/mri_violations.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# MRI Protocol Violations
+
+The MRI Violations module displays scans that have been flagged for significant variance from the study protocol.
+Use the Selection Filters to specify or search by Patient Name, MincFile, Scan Type, Project, Site, then click Show Data
+to apply the selection filters in the data table below. 
+
+The data results table contains two tabs: "Not Resolved" (displayed by default) and "Resolved". The "Resolution Status"
+colun indicates and allows updates to the scan status: Reran, Emailed Site/Pending, Inserted, etc. When a scan is updated
```
Comment (driusan) [inline modules/mri_violations/help/mri_violations.md]: There's a typo in "column".. and I find "indicates and allows updates to the scan status" is pretty awkward phraseology, but I'm not sure how to fix it.

Comment (christinerogers) [reply modules/mri_violations/help/mri_violations.md]: Unawkwarding Suggestion: @llevitis 
The "Resolution Status" column is used to display and update the scan status: ..."

File: modules/mri_violations/help/mri_violations.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# MRI Protocol Violations
```
Comment (xlecours) [inline modules/mri_violations/help/mri_violations.md]: The Menu Item is `MRI Violated Scans` should this help file show the same label/header?

**Comment by driusan** [comment]
@llevitis This is going to the `major`, not `19.0-dev` branch. It needs to be rebased to go into 19.0

**Pull request merged**